helios/stats_views.py

```diff
@@ -40,13 +40,15 @@ def elections(request):
 
   page = int(request.GET.get('page', 1))
   limit = int(request.GET.get('limit', 25))
+  q = request.GET.get('q','')
```

Comment:
ok, I'll do this.
Can I display the total of records in the navigation bar, like voters list?